http/websocket/server.go

```diff
@@ -79,34 +79,37 @@ func pushSmartCodeEvent(v interface{}) {
 		switch object := rs.Result.(type) {
 		case []*event.NotifyEventInfo:
 			evts := []bcomn.NotifyEventInfo{}
+			var contractAddrs = make(map[string]string)
 			for _, v := range object {
 				txhash := v.TxHash
 				evts = append(evts, bcomn.NotifyEventInfo{common.ToHexString(txhash[:]), v.ContractAddress.ToHexString(), v.States})
+				contractAddrs[v.ContractAddress.ToHexString()] = v.ContractAddress.ToHexString()
```

Comment:
use ` var contractAddrs = make(map[string]bool)`,  and `contractAddrs[v.ContractAddress.ToHexString()] = true` to reduce memory usage.